api/v1alpha1/etcdcluster_webhook_test.go

```diff
@@ -81,8 +89,87 @@ var _ = Describe("EtcdCluster Webhook", func() {
 				},
 			}
 			w, err := etcdCluster.ValidateCreate()
-			gomega.Expect(err).To(gomega.Succeed())
-			gomega.Expect(w).To(gomega.BeEmpty())
+			Expect(err).To(Succeed())
+			Expect(w).To(BeEmpty())
+		})
+		It("Should admit enabled empty PDB", func() {
```

Comment:
Looks like this testcase already too big. Can we test validatePdb method separately instead?